pkg/config/config.go

```diff
@@ -27,6 +27,8 @@ type Config struct {
 }
 
 type Adhoc struct {
+	AnalyticsOptOut bool `def:"false" desc:"disables analytics" mapstructure:"analytics-opt-out"`
```

Comment:
https://github.com/pyroscope-io/docs/blob/main/docs/agent-configuration-adhoc.mdx should be updated to reflect this new option.

Reply:
Done in https://github.com/pyroscope-io/docs/pull/33/commits/3e1980f6e03985e3d428780d1228eee784b6c109